plugin/githubinator.vim

```diff
@@ -94,8 +98,13 @@ endfunction
 
 vnoremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url()<CR>
 vnoremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url()<CR>
+noremap <silent> <Plug>(githubinator-open) :<C-U>call <SID>github_open_url()<CR>
+noremap <silent> <Plug>(githubinator-copy) :<C-U>call <SID>github_copy_url()<CR>
 
 if get(g:, 'githubinator_no_default_mapping', 0) == 0
     vmap <silent> gho <Plug>(githubinator-open)
     vmap <silent> ghc <Plug>(githubinator-copy)
+
+    nmap <silent> gho <Plug>(githubinator-open)
+    nmap <silent> ghc <Plug>(githubinator-copy)
```

Comment:
I realized my mistake in this commit https://github.com/danishprakash/vim-githubinator/pull/8/commits/8cde97b625d1abe0c940719b1c76e7221a2c0b82 :cry: 
xnoremap is for select mode `only` (see `:help mapmode-x`).
the right mapping is vnoremap which is used for select and visual mode

FYI, this table shows the summary.
https://github.com/mopp/dotfiles/blob/82d28d2b5acea78b3eae2351ae76010902843094/.vimrc#L127-L139